wdl-grammar/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+
+* Fixed requiring comma delimiter for `input`, `object` and `hints` literal
+  items in WDL 1.2 ([#164](https://github.com/stjude-rust-labs/wdl/pull/164)).
```

Comment:
Whoops, I just realized I was off by one for the PR number on this link. I'll correct with my next change.